Ginger/Ginger/UserControlsLib/UCTreeView/TreeView1.xaml.cs

```diff
@@ -192,5 +194,32 @@ public void AddToolbarTool(eImageType imageType, string toolTip = "", RoutedEven
         {
             //no tool bar to add to in this View type
         }
+      
+        private async void xSearchBtn_Click(object sender, RoutedEventArgs e)
+        {            
+            if (!string.IsNullOrEmpty(xSearchTextBox.Text))
+            {
+                Reporter.ToStatus(eStatusMsgKey.Searching, null, xSearchTextBox.Text);
+                Mouse.OverrideCursor = Cursors.Wait;
+                xSearchClearBtn.Visibility = Visibility.Visible;
+                xSearchBtn.Visibility = Visibility.Collapsed;
+                await Search();
+                Mouse.OverrideCursor = null;
```

Comment:
do Mouse.OverrideCursor and Reporter hide in Finally block so it will be done even if exception will be trown